app/Makefile

```
@@ -12,13 +12,19 @@ APP_NAME := main-app
 # in the event JSON is output from a process we don't log.
 DECODE_LOG := 2>&1 | python3 -u src/logging/util/decodelog.py
 
+# Required for CI flags below to work properly
+SHELL = /bin/bash -o pipefail
+
 # A few commands need adjustments if they're run in CI, specify those here
-# TODO - when CI gets hooked up, actually test this.
+# Make sure the path of the file mirrors the path to the app, or these flags will
+# not work as expected in GitHub
+## FLAKE8_FORMAT will show any findings in the "Files Changed" tab of the PR
+## MYPY_POSTPROC will show any findings in the "Files Changed" tab of the PR
 ifdef CI
  DOCKER_EXEC_ARGS := -T -e CI -e PYTEST_ADDOPTS="--color=yes"
- FLAKE8_FORMAT := '::warning file=src/%(path)s,line=%(row)d,col=%(col)d::%(path)s:%(row)d:%(col)d: %(code)s %(text)s'
+ FLAKE8_FORMAT := '::warning file=app/%(path)s,line=%(row)d,col=%(col)d::%(path)s:%(row)d:%(col)d: %(code)s %(text)s'
```

Just a small nit could we define `APP_DIR := app` above and then do
```suggestion
 FLAKE8_FORMAT := '::warning file=$(APP_DIR)/%(path)s,line=%(row)d,col=%(col)d::%(path)s:%(row)d:%(col)d: %(code)s %(text)s'
```
at some point I want to make the `app` a configurable thing to more easily support multiple apps per infra setup, and that will make it easier to catch than having app buried in the string

---

Updated the file to use the variable version of this and changed the comment to explain what the variable is and why it's needed